runtime/bcutil/jsrinliner.c

```diff
@@ -1501,7 +1501,12 @@ evaluateCodeBlock (I_32 hasRET, J9JSRICodeBlock ** blockParentPointer, U_32 star
 			break;
 
 		default:
-			if (bc > CFR_BC_jsr_w) {
+#if defined(J9VM_OPT_VALHALLA_VALUE_TYPES)
```

Comment:
I don't think this change is needed as JSRs are illegal in Java 7+ classfiles and these new bytecodes won't be in old classfiles.

Reply:
The entry point to this function is `inlineJSRs` which is only called if the method has been tagged to have a JSR.... and clearly a method with valhalla bytecodes won't